jax/lax/lax.py

```diff
@@ -2433,6 +2433,34 @@ def _conv_general_dilated_batch_rule(
       out = _reshape_axis_into(out_spec[1], out_spec[1] + 1, out)
       return out, out_spec[1]
 
+def _masked(x, logical_shape, dimensions, value=0):
```

Comment:
Could you add a brief comment or docstring explaining what this function does?

Reply:
Sure, done!